.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Feature Request
+about: Help us with new ideas
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+# Feature Request
+
+_If you have an idea or an improvement for this project that should be discussed first, please feel 
+free to open up a [discussion](https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/discussions/categories/ideas)._
+
+## What Area Will This Influence?
+_e.g., DPF, CI, build, transfer, etc._
+
+## Why Is the Feature Wanted?
+_Are there any requirements?_
+
+## Solution Proposal
+_If possible, provide a (brief!) solution proposal._
+
+## Type of Issue
+_i.e., new feature, improvement, cleanup, etc._
+
+## Checklist
+
+- [ ] assigned correct label?
+- [ ] assigned project, if necessary? (_see the section on the right-hand side -->_)
```

Comment:
I don't think non-committers can do that. Also, does that not happen automatically?

Reply:
I agree with you, will remove it.

> Also, does that not happen automatically?

No, it does not, as we could have multiple projects and decided to not have an "open issue" and "backlog" section.

Reply:
The automatism takes place as soon as an issue is initially added to the backlog.